tokens.xml

```diff
@@ -5553,6 +5634,21 @@ A card with morph can be turned face up any time for its morph cost.)</text>
             <token>1</token>
             <tablerow>2</tablerow>
         </card>
+        <card>
+            <name>Ogre Warrior Token</name>
+            <text></text>
+            <prop>
+                <colors>B</colors>
+                <type>Token Creature — Ogre Warrior</type>
+                <maintype>Creature</maintype>
+                <cmc>0</cmc>
+                <pt>4/3</pt>
+            </prop>
+            <set picURL="https://media.wizards.com/2022/snc/en_KvyeDaaJ91.png">SNC</set>
+            <reverse-related>Join the Maestros</reverse-related>
```

Comment:
would having an extra count="2" exclude="exclude" be worth having here?

Reply:
This one is almost not worth the effort. If someone has the default keybinding, they're still holding down two modifiers and could just press `T` twice.

It's more useful if the numbers have a gap, like `Gather the Townsfolk` (two or five), `Throne of Empires` (one or five), `Starnheim Unleashed` (one or X). 
(`Saproling Migration` (two or four) or `Conquerer's Pledge` (six or twelve) aren't as good as examples because they're exactly double.)

Reply:
I think it's fine to have both.

People are probably using less shortcuts as you might think and simply go with the mouse. :)